api/src/index.mjs

```diff
@@ -14,9 +14,9 @@ const apiRouter = express.Router();
 // You can delete this route once you add your own routes
 apiRouter.get("/", async (req, res) => {
   const SHOW_TABLES_QUERY =
-    process.env.DB_CLIENT === "pg"
-      ? "SELECT * FROM pg_catalog.pg_tables;"
-      : "SHOW TABLES;";
+    process.env.DB_CLIENT === "pg" ? "SELECT * FROM pg_catalog.pg_tables;" :
```

Comment:
this example route is growing a bit in complexity, i think we should just make it much more simple.